cloudinit/mainnet.yml

```diff
@@ -77,6 +77,19 @@ write_files:
     path: /etc/cron.d/sync-checker
     content: |
         */10 * * * * blockform /usr/bin/sync-checker >> /home/blockform/sync-checker.log 2>&1
+  - owner: root:root
+    path: /datadrive/geth/static-nodes.json
+    permissions: '0755'
+    content: |
+        [
+          "enode://05d8806cd230be50c779eec8eabe9fc1a8c533626dc8c7edca29c1432854b0829c6f298573cb81a21bf9f151774efb2f301239109df24aa9a52976a885f32a05@47.97.199.245:30303",
```

Comment:
@kivutar I'd be more comfortable if we fetch our bootnode from here https://github.com/ethereumjs/ethereumjs-common/blob/master/chains/mainnet.json#L60

The best will be letting the client to get its bootnodes on mainnet, it might update on every release

Reply:
I meant particularly for mainnet.  The rest is fine as is.

Reply:
as far as I understand, the static nodes must be trusted.  If so, in term of security, would it be better to use the same boot nodes?  We don't need to trust anyone else except the "official" Ethereum nodes
@hadv @kivutar wdyt?

Reply:
I am just curious if these bootnodes can be used in static-nodes.json
https://github.com/ethereum/go-ethereum/blob/461291882edce0ac4a28f64c4e8725b7f57cbeae/params/bootnodes.go#L21

Reply:
I think no. static-nodes.json is just peer nodes

Reply:
Hypothetically what would happen if the static-nodes are compromised.  They are peers, we'll be on their forked instead of the official Ethereum.
Another case that would go wrong is that the static-nodes just forget to update their clients in the event of hard fork.
Would it be much safer to stay with Ethereum official nodes?

@hadv  do you know who own these static nodes?

Reply:
For mainnet, using random static-nodes has more downsides than upsides IMO.
I don't care much about testnets

Reply:
Maybe I am missing something. @hadv can you please elaborate the "just peer nodes"?  What damage could peer nodes do?

Reply:
Yes, basically your concerns about compromised peer nodes are true; especially on the mainnet. I think we need to select peer nodes for mainnet carefully if we use. You're right; if we dont have trusted peer nodes then it'd better to use default officially bootnodes. 

Btw, I have 2 things to make it more clearly

1. Using static-nodes does not mean we dont using the default bootnodes. Then the default bootnodes are still used in our node.
2. I'm not sure that the default officially bootnodes are maintain and update correctly or not?

*Conclusion*:
- we dont use static-nodes for mainnet if we dont know any trusted peers.
- use only default mainnet bootnodes; if our node cannot sync with the mainnet then we need to find other bootnodes or trusted peers.

Reply:
> Maybe I am missing something. @hadv can you please elaborate the "just peer nodes"? What damage could peer nodes do?

okay, maybe you're right! we can add both peer nodes and bootnodes to the  static-node. I need to investigate more.

Reply:
sounds like a good plan.  By "attack/compromised" I meant most of them are hypothetical scenarios like front running attack, selfish miner attack...  I agree that It's not practical to deal damage to Ethereum blockchain now.

However, that our "trusted" peers forget to upgrade their geth clients when there is hard fork is more valid issue.